.github/workflows/ci.yml

```diff
@@ -45,11 +45,11 @@ jobs:
           npm run lint:deps
 
       - name: Add nonce for parallel tests
-        run: echo ::set-env name=PERCY_PARALLEL_NONCE::`date +%s`
+        run: echo "PERCY_PARALLEL_NONCE=`date +%s`" >> $GITHUB_ENV
```

Comment:
now that we're using `percy exec`, I wonder if we actually still need the `PERCY_PARALLEL_NONCE` 🤔